Programming/Java/Inheritance/Inher.java

```diff
@@ -0,0 +1,50 @@
+class Member
+{
+	String name,address,phno;
+	int age,salary;
+	void printSalary()
+	{
+		System.out.println("Salary = "+salary);
+	}
+};
+class Employee extends Member
+{
+	String specialization;
+	void display()
+	{
+		System.out.println(name+"\t"+specialization+"\t"+address+"\t"+age+"\t"+phno);
+	}	
+};
+class Manager extends Member
+{
+	String department;
+	void display()
+	{
+		System.out.println(name+"\t"+department+"\t"+address+"\t"+age+"\t"+phno);
+	}
+};
+class Inher
+{
+	public static void main()
+	{
+		Employee e=new Employee();
+		e.name="Samuel";
+		e.age=34;
+		e.phno="767985876";
+		e.adress="4/345,Jopling road";
+		e.salary=40000;
+		e.dispaly();
+		e.printSalary();
+	
+		Manager m=new Manager();
+		m.name="Joshua";
+		m.age=39;
+		m.phno="631890755";
+		m.adress="5-676,Rajendrapur Rd";
+		m.salary=200000;
+		m.dispaly();

```

Comment:
@kashish-goel it seems you have missed-spelled the word ```adress``` it should be ```address``` 
Also, ```e.dispaly()```  should be ```e.display()```
Same as before so it's a humble request please go through your code before raising a PR 😅.